README.md

```diff
@@ -862,7 +862,7 @@ Annotations for function definitions should be as general as possible.
 
 ```julia
 # Yes:
-splicer(arr::AbstractArray, step::Integer) = arr[1:step:end]
+splicer(arr::AbstractArray, step::Integer) = arr[begin:step:end]
 
 # No:
 splicer(arr::Array{Int}, step::Int) = arr[1:step:end]
```

Comment:
Should keep these consistent as we want to draw attention to the types used in dispatch and not the `1`:
```suggestion
splicer(arr::Array{Int}, step::Int) = arr[begin:step:end]
```